src/test/java/com/meilisearch/sdk/TestUtils.java

```diff
@@ -0,0 +1,53 @@
+package com.meilisearch.sdk;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.nio.file.Files;
+import java.lang.reflect.Type;
+import java.util.List;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+public class TestUtils {
+
+	String movies_data;
+	Movie[] movies = new Movie[10];
+	Gson gson = new Gson();
```

Comment:
more of a general point: unless you really need fields package private, just declare them as private. 
helps to avoid other parts of the program fucking around with the internal state of the class.